build.gradle

```diff
@@ -19,6 +19,7 @@ allprojects {
     version = '0.1'
 
     apply plugin: "scala"
+    apply plugin: 'cz.alenkacz.gradle.scalafmt'
```

Comment:
this is correct! We have to apply that plugin to every module so the tasks are available in that module

Reply:
Thanks!